SolarEnergy/Aero/waypoints.py

```diff
@@ -0,0 +1,135 @@
+import geopy
+import geopy.distance
+from geographiclib.geodesic import Geodesic
+import itertools
+import csv
+from datetime import datetime, timedelta
+
+gDEBUG = True
+
+class FlightPath():
+
+    def __init__(self, iniday=None, initime=None, inipos=None):
+        self.iniday = iniday
+        self.initime = initime
+        self.inipos = inipos
+        self.waypoints = [inipos]
+        self.iwaypoints = None
+
+    def GenIData(self):
+        # Mark original waypoints as 1 (at some point I think it'll be best to redo this a bit so it's less clunky)
```

Comment:
I appreciate you probably still want to redo this, but maybe it shouldn't be noted in a code comment if we're making this public. Or at least, word it a bit differently.